web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :screen_name, :string
+    field :profile_image_url, :string
+    field :twitter_id, :integer
+    field :name, :string
```

Comment:
Like I said above, I don't think it's super clear what these fields are - I sort of had to read the entire thing to make sense of them.

1. If `twitter_id` is what we use to populate the user id, should that field just be `id`?
2. I'm more in favour of `user_name` and `display_name` over `screen_name` and `name`.